protocol/xmrmaker/claim.go

```diff
@@ -60,9 +61,19 @@ func (s *swapState) claimFunds() (*ethtypes.Receipt, error) {
 		sc := s.getSecret()
 		receipt, err = s.sender.Claim(s.contractSwap, sc)
 		if err != nil {
-			return nil, err
+			if strings.Contains(err.Error(), "insufficient funds for gas * price + value") {
```

Comment:
We can update this to `core.ErrInsufficientFunds` from `go-ethereum` if we want to avoid the message contents check.